pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -970,3 +972,36 @@ message StatusResponse {
     // Status of the Runtime.
     RuntimeStatus status = 1;
 }
+
+message ImageFsInfoRequest {}
+
+// UInt64Value is the wrapper of uint64.
+message UInt64Value {
+    // The value.
+    uint64 value = 1;
+}
+
+// FsInfo is the filesystem information of the images.
+message FsInfo {
+    // The block device name associated with the filesystem.
+    string device = 1;
+    // Path where the filesystem is mounted.
+    string mountpoint = 2;
+    // Filesystem usage in bytes.
+    uint64 capacity = 3;
+    // Bytes available for non-root use.
+    uint64 available = 4;
+    // Number of bytes used on this filesystem.
+    uint64 usage = 5;
+    // Labels associated with this filesystem.
+    repeated string labels = 6;
+    // Number of Inodes.
+    UInt64Value inodes = 7;
```

Comment:
Why using `UInt64Value`? Is this optional? If inode information *is* optional, I'd suggest grouping them into a separate message.

Also rename the fields to:
`inodes_used`
`inodes_available`
`inodes_free`